atomic_reactor/plugins/input_osv3.py

```diff
@@ -112,9 +115,9 @@ def has_plugin(self, phase, target_plugin):
                 self.remove_plugin(phase, PLUGIN_PULP_PULL_KEY, 'no pulp or koji available')
             else:
                 has_pulp_pull |= has_plugin(self, phase, PLUGIN_PULP_PULL_KEY)
-        arrangement_six = self.plugins_json.get('arrangement_version', 0) >= 6
-        orchestrator_build = self.plugins_json.get('build_type', None) == 'orchestrator'
-        if arrangement_six and orchestrator_build:
+
+        # if we're here at all, we're on arrangment 6+
+        if self.build_type == 'orchestrator':
```

Comment:
I'd rather we remove this block completely. It's really unnecessary. Let the plugin itself decide whether or not it should run.